sky/optimizer.py

```diff
@@ -1293,9 +1305,10 @@ def _fill_in_launchable_resources(
                        if resources.cloud is not None else enabled_clouds)
         # If clouds provide hints, store them for later printing.
         hints: Dict[clouds.Cloud, str] = {}
-        for cloud in clouds_list:
-            feasible_resources = cloud.get_feasible_launchable_resources(
-                resources, num_nodes=task.num_nodes)
+
+        feasible_list = subprocess_utils.maybe_parallelize_cloud_operation(
+            _make_resource_finder(resources, task.num_nodes), clouds_list)
```

Comment:
wondering if we can simplify this with `run_in_parallel(lambda cloud: cloud.get_feasible_launchable_resources(resources, num_nodes), cloud_list)`

Reply:
I tried lambda at the very beginning but got a pylint warning😂:

```
sky/optimizer.py:1311:16: W0640: Cell variable resources defined in loop (cell-var-from-loop)
```

Reply:
Just figured out how to avoid potential closure issue in loop when using lambda https://stackoverflow.com/questions/25314547/cell-var-from-loop-warning-from-pylint

Reply:
Ahh, good catch. Just wondering if we can make the function a bit simpler.